.gitignore

```diff
@@ -1,2 +1,25 @@
 node_modules/
```

Comment:
Hmm, I'm just unfamiliar - why was this included in a git push if it's a hidden file?